RenderStatePackagerLib/generated/DeviceObjectDescriptorParserParser.hpp

```diff
@@ -0,0 +1,230 @@
+/*
```

Comment:
Why is there DeviceObjectDescriptorParser.hpp in both generated and include fodlers?

Reply:
This is `DeviceObjectDescriptorParserParser.hpp`.  This file has structures for which we generate `Serialize` and `Deserialize`